### PR TITLE
delete job

### PR DESCRIPTION
**Pull request by utas-tajona**
backend & frontend

### CODE REVIEW

**Comment by utas-tajona** [comment]
https://github.com/aodn/backlog/issues/2640

File: api/src/main/java/au/org/aodn/nrmn/restapi/model/db/StagedJobLog.java [hunk 1]
```diff
@@ -37,7 +25,7 @@
     @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "staged_job_log_id_seq")
     private Long id;
 
-    @ManyToOne
+    @ManyToOne(cascade = CascadeType.ALL)
```
Comment (jonescc) [inline api/src/main/java/au/org/aodn/nrmn/restapi/model/db/StagedJobLog.java:28]: Did you need this? I don't think its required. Normally cascade is used on the parent only.

File: api/src/main/java/au/org/aodn/nrmn/restapi/model/db/StagedRow.java [hunk 2]
```diff
@@ -111,7 +112,7 @@
     private Map<Integer, String> measureJson;
 
     @JsonIgnore
-    @ManyToOne(fetch = FetchType.LAZY)
+    @ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
```
Comment (jonescc) [inline api/src/main/java/au/org/aodn/nrmn/restapi/model/db/StagedRow.java:115]: And here.

**Review by jonescc — COMMENTED** [review]
LGTM.  @nspool ?

**Review by nspool — COMMENTED** [review]
The JS style seems to differ from the .prettierrc config in the UI project. Can you please check that it is using the right config?

**Comment by utas-tajona** [comment]
i've setup prettier on vscode and run it on the js file of this PR

**Review by nspool — COMMENTED** [review]
Looks like it's matching the config.

**Pull request merged**